bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/internal/WebClientFactoryImpl.java

```diff
@@ -350,8 +384,32 @@ private SslContextFactory createSslContextFactory(@Nullable String endpoint) {
                 }
             }
         }
+
         String excludeCipherSuites[] = { "^.*_(MD5)$" };
         sslContextFactory.setExcludeCipherSuites(excludeCipherSuites);
         return sslContextFactory;
     }
+
+    @Reference
+    protected void setExtensibleTrustManager(ExtensibleTrustManager extensibleTrustManager) {
+        this.extensibleTrustManager = extensibleTrustManager;
+    }
+
+    protected void unsetExtensibleTrustManager(ExtensibleTrustManager extensibleTrustManager) {
+        if (this.extensibleTrustManager == extensibleTrustManager) {
+            this.extensibleTrustManager = null;
+        }
+    }
+
+    @Reference(service = TrustManagerProvider.class, cardinality = ReferenceCardinality.OPTIONAL, policy = ReferencePolicy.DYNAMIC)
+    protected void setTrustmanagerProvider(TrustManagerProvider trustmanagerProvider) {
```

Comment:
Sure, this method is called by the framework only, but should we mark it deprecated (and the trustmanagerProvider member of that class, too)?
Just to signal the readers of that class, that the `TrustManagerProvider` usage is outdated?

Reply:
done